apps/Learning-Buddy/app.config.js

```diff
@@ -5,6 +5,7 @@ module.exports = {
   extra: {
     IP: process.env.IP,
     supabaseURL:process.env.SUPABASEURL,
-    supabaseKEY:process.env.SUPABASEKEY
+    supabaseKEY_PUBLIC:process.env.SUPABASEKEY_PUBLIC,
+    supabaseKEY_SECRET:process.env.SUPABASEKEY_SECRET
```

Comment:
As I put in the mention section for this PR, I don't know why it would be in the frontend; I'm not sure if there's a sensible business case for it, but we can always remove it down the line if we so choose. It's not being exposed anywhere at present time as it's in dotenv; I did remove it from app.config.js for the time being.